### ISSUE TITLE
jpass doesn't work with java9

### ISSUE THREAD

**Issue by CatPlanet** [gaborbata/jpass#7]
https://i.imgur.com/gEQEuvy.png

This is what I get when I try to use passfile.

**Comment by gaborbata** [comment]
Thanks @CatPlanet for reporting the issue. `java.xml.bind` is marked as deprecated for Java 9. I will resolve the issue when I have time. Until that, as a workaround, you can use the `--add-modules` option with Java 9 to start the application, e.g.:

```
java -jar --add-modules=java.xml.bind jpass-0.1.15-RELEASE.jar
```

Some related materials:
* https://docs.oracle.com/javase/9/docs/api/java.xml.bind-summary.html
* https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j
* https://www.concretepage.com/forum/thread?qid=531

**Comment by gaborbata** [comment]
The issue has been fixed in v0.1.16 release (https://github.com/gaborbata/jpass/pull/8):
* https://github.com/gaborbata/jpass/releases/tag/v0.1.16
